sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.validateOffering.e2e-spec.ts

```diff
@@ -757,6 +758,389 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-validateOffering",
+      yearOfStudy: 1,
+      offeringIntensity: OfferingIntensity.fullTime,
+      offeringDelivered: OfferingDeliveryOptions.Online,
+      isAviationOffering: OfferingYesNoOptions.Yes,
+      aviationCredentialType:
+        AviationCredentialTypeOptions.PrivatePilotTraining,
+      hasOfferingWILComponent: "no",
+      studyStartDate: "2024-06-01",
+      studyEndDate: "2024-12-02",
+      lacksStudyBreaks: true,
+      studyBreaks: [],
+      offeringType: OfferingTypes.Public,
+      offeringDeclaration: true,
+      actualTuitionCosts: 1234,
+      programRelatedCosts: 3211,
+      mandatoryFees: 456,
+      exceptionalExpenses: 555,
+      courseLoad: 30,
+    };
+    const endpoint = `/institutions/education-program-offering/location/${collegeFLocation.id}/education-program/${collegeFFullTimeProgram.id}/validation`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        offeringStatus: OfferingStatus.CreationPending,
+        errors: [],
+        infos: [],
+        warnings: [
+          {
+            typeCode: "programNotAviation",
+            message:
+              "Aviation Offering is defined as yes but the program is not an aviation program.",
+          },
+          {
+            typeCode: "programAviationCredentialMismatch",
+            message:
+              "Aviation credential type for the program does not match the offering aviation credential.",
+          },
+          {
+            typeCode: "aviationCredIsPrivatePilotTraining",
+            message:
+              "aviationCredentialType must be one of the following values: commercialPilotTraining, instructorsRating, endorsements",
+          },
+        ],
+        studyPeriodBreakdown: {
+          fundedStudyPeriodDays: 185,
+          totalDays: 185,
+          totalFundedWeeks: 27,
+          unfundedStudyPeriodDays: 0,
+        },
+      });
+  });
+
+  it.only("Should return warning when a part-time program is an aviation program and has an aviation offering with a non matching aviation credential type.", async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFPartTimeProgram.isAviationProgram = "yes";
+    collegeFPartTimeProgram.credentialTypesAviation = {
+      commercialPilotTraining: true,
+      instructorsRating: true,
+      endorsements: false,
+      privatePilotTraining: false,
+    };
+    await db.educationProgram.save(collegeFPartTimeProgram);
+    const payload = {
+      offeringName: "Second aviation offering validation",
+      yearOfStudy: 1,
+      offeringIntensity: OfferingIntensity.fullTime,
+      offeringDelivered: OfferingDeliveryOptions.Onsite,
+      isAviationOffering: OfferingYesNoOptions.Yes,
+      aviationCredentialType:
```

Comment:
Can the `endorsements` be used instead to trigger only the `programAviationCredentialMismatch` warning that is the target of this E2E?